FLAIR.md

```diff
@@ -0,0 +1,29 @@
+Acknowledgments
+=========
+
+<p valign="middle"><a href="https://www.browserstack.com/"><img src="http://media.archiveofourown.org/ao3/logos/Browserstack.svg" width="200"/></a> for cross-browser testing tool. </p>
```

Comment:
If you want to vertically align it so the text and the middle of the image are lined up, the `valign` needs to be on the image tag.